src/createServer.js

```diff
@@ -1,3 +1,50 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+'use strict';
+
+const http = require('http');
+const { getParamsOfUrl } = require('./getParamsOfUrl');
+
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const [text, toCase] = getParamsOfUrl(req);
+
+    const errors = [];
+
+    /* eslint-disable */
+    if (!text) {
+      errors.push({ message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".' });
+    }
+
+    if (!toCase) {
+      errors.push({ message: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".' });
+    } else if (!['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'].includes(toCase)) {
+      errors.push({ message: `This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.` });
+    }
+    /* eslint-enable */
+
+    if (errors.length > 0) {
+      res.writeHead(400, { 'Content-Type': 'application/json' });
```

Comment:
DRY
make a separate function for send response

Reply:
sorry, dont understand this comment😓

Reply:
You did it 2 times
Make a separate function to avoid it
```js
res.writeHead(..., { 'Content-Type': 'application/json' });
res.end(JSON.stringify({}))
```